arrows/colmap/colmap.cpp

```diff
@@ -0,0 +1,11 @@
+#include <arrows/colmap/colmap.h>
+#include <colmap/mvs/image.h>
```

Comment:
Compilation fails for me here. This should be resolved by dealing with Matt's comment about finding the colmap library.